docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -97,16 +77,29 @@ down by HTTP route.
 This might let you determine that `/api/sad` had a higher rate of erroring
 spans than `/api/happy`, for example.
 
-### The `count_over_time`, `min_over_time` and `max_over_time` functions
+## The `count_over_time` function
 
 The `count_over_time()` let you counts the number of matching spans per time interval.
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
+
+
+### Example
 
 ```
 { name = "GET /:endpoint" } | count_over_time() by (span.http.status_code)
 
 ```
 
+## The `min_over_time` function
+
 The `min_over_time()` let you aggregate numerical values by computing the minimum value of them, such as the all important span duration.
```

Comment:
```suggestion
The `min_over_time()` lets you aggregate numerical attributes by calculating their minimum value. For example, you could choose to calculate the minimum duration of a group of spans, or you could choose to calculate the minimum value of a custom attribute you've attached to your spans, like `span.shopping.cart.entries`. 
```

Reply:
Maybe that's a crappy example.

Reply:
I think it's good to have an example. It's a start and we can iterate if we want to modify it later.